pkg/config/cloud/powervs/powervs_regions.go

```diff
@@ -0,0 +1,102 @@
+package powervs
+
+import (
+	"fmt"
+)
+
+// Since there is no API to query these, we have to hard-code them here.
+
+// Region describes resources associated with a region in Power VS.
+// We're using a few items from the IBM Cloud VPC offering. The region names
+// for VPC are different so another function of this is to correlate those.
+type Region struct {
+	Description string
+	VPCRegion   string
+	Zones       []string
+}
+
+// Regions holds the regions for IBM Power VS, and descriptions used during the survey.
```

Comment:
nit: "the survey" is an IPI term. I change this to:

"Regions provides a mapping between Power VS and IBM Cloud VPC regions."

Reply:
Fixed.